frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDashboard.tsx

```diff
@@ -13,9 +17,12 @@ const TeamEventCreditDashboard = (props: {
   approvedAttendance: TeamEventAttendance[];
   pendingAttendance: TeamEventAttendance[];
   isAttendanceLoading: boolean;
+
```

Comment:
See my comment below, but if you were to add `setPendingAttendance` as a prop to this component, you'd do it here.

```
approvedAttendance: TeamEventAttendance[];
pendingAttendance: TeamEventAttendance[];
isAttendanceLoading: boolean;
// insert the following line
setPendingAttendance: Dispatch<SetStateAction<TeamEventAttendance[]>>;
```